tiledb/sm/config/config.cc

```diff
@@ -561,17 +563,55 @@ Config::~Config() = default;
 /*                API             */
 /* ****************************** */
 
+void Config::login() {
+  // Retrieve $HOME path
+  std::string home_dir;
+#ifdef _WIN32
+  home_dir =
+      std::string(std::getenv("HOMEDRIVE")) + std::string(getenv("HOMEPATH"));
+#else
+  home_dir = std::string(std::getenv("HOME"));
+#endif
+
+  // For library versions 2.27.0 and older, simply parse the local .json file
+  auto version = constants::library_version;
+  if (version[0] <= 2 && version[1] <= 27) {
```

Comment:
I don't understand this, `dev` will soon switch to 2.28 and and this will always be false.